src/main.cpp

```diff
@@ -430,6 +519,31 @@ int main(const int argc, const char *argv[])
     inset_state.destroy_fftw_plans_for_rho();
     inset_state.ref_to_rho_init()->free();
     inset_state.ref_to_rho_ft()->free();
+
+    // Calculate distances
+    std::map<std::string, double> hausdorff_distances =
+      inset_state.get_geo_div_differences(hausdorff_distance, true);
+    double total_distance = 0;
+    std::cout << "\nHausdorff distances:\n";
+    for (const auto &[key, val] : hausdorff_distances) {
+      std::cout << key << ": " << val << std::endl;
+      total_distance += val;
+    }
+    std::cout << "\nAverage Hausdorff distance: "
+              << total_distance / double(hausdorff_distances.size())
+              << std::endl;
+
+    std::map<std::string, double> symmetric_distances =
+      inset_state.get_geo_div_differences(symmetric_distance, true);
+    total_distance = 0;
+    std::cout << "\nSymmetric distances:\n";
+    for (const auto &[key, val] : symmetric_distances) {
+      std::cout << key << ": " << val << std::endl;
+      total_distance += val;
+    }
+    std::cout << "\nAverage symmetric distance: "
+              << total_distance / double(symmetric_distances.size())
+              << std::endl;
```

Comment:
I don't think the impact on runtime is that great, and these distances are only calculated after we stop the timers.

Reply:
I agree there will not be a significant increase in runtime. But, this part of the code will always execute and the resultant information will be shown. And, it is a lot of lines of information since it is printing information for every GeoDiv twice. Imagine the console after generating a cartogram for a country that has 50+ GeoDivs. That's why I feel it is best if we have a flag to output this information.

Reply:
Also, I do not think 

> these distances are only calculated after we stop the timers.

is true. We stop the timer on line 557 which is certainly after the distance calculations.